src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -1,12 +1,17 @@
 # Changelog
 
-## Unreleased
+## 1.8.0-rc.1
```

Comment:
Why RC instead of beta?

Reply:
We can do RC as we do not expect significant changes in exporter. And it is consistent with the SDK version (1.8.0-rc.1) it depends on.

Reply:
Why don’t we release a stable version then? 
If this release is targeted for `PrivatePreviewEnableOtlpProtobufEncoding`, I recommend a beta release to gather feedback.

Reply:
> Why don’t we release a stable version then?

We cannot, we need at least one pre-release version for customers relying on experimental APIs (e.g. exemplars). That is the reason to update SDK dependency as well to RC version.

> If this release is targeted for PrivatePreviewEnableOtlpProtobufEncoding, I recommend a beta release to gather feedback

The decision to apply the RC or Beta tag is not determined by the `PrivatePreviewEnableOtlpProtobufEncoding` option. The exporter's current state is close to stable because no changes have been made to existing APIs or functionality. Thus, the RC tag is appropriate here.